source/optional_features/custom_metadata/index.html.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+title: Adding custom metadata
+weight: 135
+---
+
+# Adding custom metadata
+
+You can add custom metadata to a new payment. For example, you can add a reference number from your finance or accounting system, so you can reconcile the payment later.
+
+You add metadata when you make an API call to create a new payment. You cannot add metadata to [Direct Debit transactions](/direct_debit/#direct-debit) or [payment links](/payment_links/#payment-links).
+
+End users cannot see metadata while they're making a payment.
+
+## Add metadata to a payment
+
+Include a `metadata` object in the body of your [Create new payment](/making_payments/#making-payments) request, for example:
+
+```javascript
+"metadata": {
+   "ledger_code": "AB100",
+   "an_internal_reference_number": 200,
+...
+}
+```
+
+The `metadata` object must contain between 1 and 10 parameters.
+
+Each parameter key must be a unique string between 1 and 30 characters long. If a key is not unique, the API will remove all but one of the duplicate keys from the `metadata` object.
```

Comment:
@oswaldquek This is all ready to merge now. Before we merge, could you review this paragraph - just wanted to double-check it's still accurate in its post-2i form.

Reply:
How about: `Each parameter key must be a unique string between 1 and 30 characters long. Non unique keys will be arbitrarily removed.`?